pkg/app/config.go

```diff
@@ -258,4 +258,5 @@ type InitConfigProvider interface {
 // when enable reuse-values, reuse the last release's values and merge in any overrides values.
```

Comment:
add more comments.

Reply:
Ive just made this comment more adequate now